packages/neos-ui-guest-frame/src/manifest.js

```diff
@@ -0,0 +1,98 @@
+import {Map} from 'immutable';
+import {$get} from 'plow-js';
+
+import manifest from '@neos-project/neos-ui-extensibility';
+import {SynchronousRegistry} from '@neos-project/neos-ui-extensibility/src/registry';
+import {selectors} from '@neos-project/neos-ui-redux-store';
+
+import makeInitializeGuestFrame from './initializeGuestFrame';
+
+import initializeContentDomNode from './initializeContentDomNode';
+import {
+    findNodeInGuestFrame,
+    findAllOccurrencesOfNodeInGuestFrame,
+    createEmptyContentCollectionPlaceholderIfMissing
+} from './dom';
+import InlineUI from './InlineUI';
+
+import style from './style.css';
+
+manifest('@neos-project/neos-ui-guestframe', {}, globalRegistry => {
+    const guestFrameRegistry = new SynchronousRegistry(`
+        # Registry for guest-frame specific functionalities
```

Comment:
Please document the structure for makeInitializeGuestFrame and InlineUIComponent; with a short sentence what these do, and what their API is...